app/models/BusinessType.scala

```diff
@@ -53,6 +56,15 @@ object BusinessType extends Enumerable.Implicits {
     Radios(field, items)
   }
 
+  def options(implicit messages: Messages): Seq[RadioItem] = values.zipWithIndex.map {
```

Comment:
I can't see what this method is for can you explain?

Reply:
As far as I can understand this method is to identify the businessType based upon the choosen option . Not sure I might be wrong.